src/documents/views/StudyFilesListView.js

```diff
@@ -76,6 +77,15 @@ const StudyFilesListView = ({
   const files = !loading ? studyByKfId.files.edges : [];
   return (
     <Grid as={Segment} basic container columns={1}>
+      <Helmet>
+        {studyByKfId ? (
+          <title>
+            KF Data Tracker - Study documents for {studyByKfId.name}
+          </title>
+        ) : (
+          <title>KF Data Tracker - Study documents</title>
+        )}
+      </Helmet>
```

Comment:
Could _maybe_ compact this down to, but would test it first as I'm not sure if it would bork or not. Also may want to use the study id here as it's shorter than the name
```js 
<Helmet>
<title>
KF Data Tracker - Study documents `${studyByKfId ? 'for '+studyByKfId.name : null}`
</title>
</Helmet>
```